challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -15319,6 +15319,1381 @@
+        },
+        {
+          "text":
+            "assert.equal(repString(\"1110111011\"), \"1110\", '<code>repString(\"1110111011\")</code> should return <code>\"1110\"</code>.'",
+          "testString":
+            "assert.equal(repString(\"1110111011\"), \"1110\", '<code>repString(\"1110111011\")</code> should return <code>\"1110\"</code>.');"
+        },
+        {
+          "text":
+            "assert.equal(repString(\"0010010010\"), \"001\", '<code>repString(\"0010010010\")</code> should return <code>\"001\"</code>.'",
+          "testString":
+            "assert.equal(repString(\"0010010010\"), \"001\", '<code>repString(\"0010010010\")</code> should return <code>\"001\"</code>.');"
+        },
+        {
+          "text":
+            "assert.equal(repString(\"1010101010\"), \"1010\", '<code>repString(\"1010101010\")</code> should return <code>\"1010\"</code>.'",
+          "testString":
+            "assert.equal(repString(\"1010101010\"), \"1010\", '<code>repString(\"1010101010\")</code> should return <code>\"1010\"</code>.');"
+        },
+        {
+          "text":
+            "assert.equal(repString(\"1111111111\"), \"11111\", '<code>repString(\"1111111111\")</code> should return <code>\"11111\"</code>.'",
+          "testString":
+            "assert.equal(repString(\"1111111111\"), \"11111\", '<code>repString(\"1111111111\")</code> should return <code>\"11111\"</code>.');"
+        },
+        {
+          "text":
+            "assert.equal(repString(\"0100100\"), \"010\", '<code>repString(\"0100100\")</code> should return <code>\"010\"</code>.'",
+          "testString":
+            "assert.equal(repString(\"0100100\"), \"010\", '<code>repString(\"0100100\")</code> should return <code>\"010\"</code>.');"
+        },
+        {
+          "text":
+            "assert.equal(repString(\"11\"), \"1\", '<code>repString(\"11\")</code> should return <code>\"1\"</code>.'",
+          "testString":
+            "assert.equal(repString(\"11\"), \"1\", '<code>repString(\"11\")</code> should return <code>\"1\"</code>.');"
+        },
+        {
+          "text":
+            "assert.equal(repString(\"00\"), \"0\", '<code>repString(\"00\")</code> should return <code>\"0\"</code>.'",
+          "testString":
+            "assert.equal(repString(\"00\"), \"0\", '<code>repString(\"00\")</code> should return <code>\"0\"</code>.');"
+        }
+      ],
+      "id": "5a23c84252665b21eecc7fb1",
+      "challengeType": 5,
+      "releasedOn": "September 6, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": ["function repString (str) {", "  // Good luck!", "}"],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Resistor mesh",
+      "description": [
+        "<a href=\"http://rosettacode.org/wiki/image:resistor-mesh.svg\">300px||right</a>",
```

Comment:
`"Given   10&times;10   grid nodes  (as shown in the image)  interconnected by   1Ω   resistors as shown, write a function to find the resistance between point  <b>A</b>  and  <b>B</b>.`

Also, is there any way to get the image this challenge is referring to?